### PR TITLE
chore: remove healthcheck code

### PR DESCRIPTION
**Pull request by hazzadous**
It's not used anywhere. We're rather using healthcheck http probes.

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: plugin-server/src/main/pluginsServer.ts [hunk 1]
```diff
@@ -335,13 +333,6 @@ export async function startPluginsServer(
             schedule.scheduleJob('*/5 * * * *', async () => {
                 await piscina?.broadcastTask({ task: 'reloadAllActions' })
             })
-            // every 5 seconds set Redis keys @posthog-plugin-server/ping and @posthog-plugin-server/version
-            schedule.scheduleJob('*/5 * * * * *', async () => {
-                await hub!.db!.redisSet('@posthog-plugin-server/ping', new Date().toISOString(), 60, {
```
Comment (xvello) [inline plugin-server/src/main/pluginsServer.ts:340]: That redis key is used by [the app's preflight logic](https://github.com/PostHog/posthog/blob/6f29a9c0347ae792b2a5509a6fae5a6db3e17c22/posthog/utils.py#L782-L787). We need to keep it or change the preflight to check for another condition.

**Review by xvello — CHANGES_REQUESTED** [review]

File: plugin-server/src/main/pluginsServer.ts (hunk 1 above)
Comment (hazzadous) [reply plugin-server/src/main/pluginsServer.ts:340]: ah damn I'd missed that 👍

Comment (hazzadous) [reply plugin-server/src/main/pluginsServer.ts:340]: laaaaaaame

Comment (hazzadous) [reply plugin-server/src/main/pluginsServer.ts:340]: Have added that back 😓

**Review by xvello — APPROVED** [review]
LGTM! As an extra precaution, could you try resetting your local devenv and going through the preflight? Let's not break hobby deploy the week after we fix it ^^

**Pull request merged**